.nvmrc

```diff
@@ -1 +1 @@
-v10.4.1
+10.5.0
```

Comment:
This seems to work fine as `10.5.0` is "versionish", however the canonical would be `v10.5.0`, which is the output of `node --version`.